src/components/snack-bar/CustomSnackbar.js

```diff
@@ -0,0 +1,48 @@
+import Snackbar from '@material-ui/core/Snackbar/Snackbar';
```

Comment:
And you use `@material-ui` here.

Reply:
That is where `@material-ui` was used, you realize this `PR` is `11 days` old. IT was two weeks ago when this was done. But I can still fix that Let me fix in a few minutes. In my next commit amendment, this will be fixed.